gremlin-driver/src/test/java/org/apache/tinkerpop/gremlin/driver/ClusterBuilderTest.java

```diff
@@ -63,6 +63,9 @@ public static Iterable<Object[]> data() {
                 {"nioPoolSize0", Cluster.build().nioPoolSize(0), "nioPoolSize must be greater than zero"},
                 {"nioPoolSizeNeg1", Cluster.build().nioPoolSize(-1), "nioPoolSize must be greater than zero"},
                 {"connectionSetupTimeoutMillis0", Cluster.build().connectionSetupTimeoutMillis(0), "connectionSetupTimeoutMillis must be greater than zero"},
+                {"connectionSetupTimeoutMillisLtMax", Cluster.build().connectionSetupTimeoutMillis(15000).maxWaitForConnection(1000), "maxWaitForConnection cannot be less than connectionSetupTimeoutMillis"},
+                {"sslHandshakeTimeoutMillisNeg1", Cluster.build().sslHandshakeTimeoutMillis(-1), "sslHandshakeTimeoutMillis must be greater than or equal to zero"},
+                {"sslHandshakeTimeoutMillisLtConnectionSetup", Cluster.build().sslHandshakeTimeoutMillis(15000).connectionSetupTimeoutMillis(10000), "connectionSetupTimeoutMillis cannot be less than sslHandshakeTimeoutMillis"},
```

Comment:
nit: same error message logic ordering thing
```suggestion
                {"connectionSetupTimeoutMillisLtMax", Cluster.build().connectionSetupTimeoutMillis(15000).maxWaitForConnection(1000), "connectionSetupTimeoutMillis cannot be greater than maxWaitForConnection"},
                {"sslHandshakeTimeoutMillisNeg1", Cluster.build().sslHandshakeTimeoutMillis(-1), "sslHandshakeTimeoutMillis must be greater than or equal to zero"},
                {"sslHandshakeTimeoutMillisLtConnectionSetup", Cluster.build().sslHandshakeTimeoutMillis(15000).connectionSetupTimeoutMillis(10000), "sslHandshakeTimeoutMillis cannot be greater than connectionSetupTimeoutMillis"},
```